code/modules/mob/living/carbon/xenomorph/castes/hivemind/hivemind.dm

```diff
@@ -19,22 +19,23 @@
 	tier = XENO_TIER_ZERO
 	upgrade = XENO_UPGRADE_ZERO
 
-	mouse_opacity = MOUSE_OPACITY_OPAQUE
+	mouse_opacity = MOUSE_OPACITY_TRANSPARENT
 	lighting_alpha = LIGHTING_PLANE_ALPHA_MOSTLY_INVISIBLE
 	see_invisible = SEE_INVISIBLE_LIVING
-	invisibility = INVISIBILITY_OBSERVER
+	invisibility = INVISIBILITY_MAXIMUM
```

Comment:
Any particular reason why observers shouldn't be able to see this?

Reply:
INVISIBILITY_OBSERVER and INVISIBILITY_OPAQUE (the two flags before) would cause things like bodyblocking to occur. these two flag changes fixed hiveminds bodyblocking (it seems from testing), SADAR's being able to collide with hiveminds, marines being able to attack hiveminds etc.

Reply:
Looking into this further, can i get more details on how this happens?

hiveminds can't be hit by projectiles.
```
/mob/living/carbon/xenomorph/hivemind/projectile_hit()
	return FALSE
```

Reply:
Im not sure either, but SADAR missiles would trigger on them, and they could be pointblanked by shotguns and normal weapons/meeled.

To be honest, im not sure what was causing it but its fixed